Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -822,8 +825,37 @@ noncomputable instance Real.isROrC : IsROrC ℝ where
   norm_sq_eq_def_ax z := by simp only [sq, Real.norm_eq_abs, ← abs_mul, abs_mul_self z, add_zero,
     mul_zero, AddMonoidHom.zero_apply, AddMonoidHom.id_apply]
   mul_im_I_ax z := by simp only [MulZeroClass.mul_zero, AddMonoidHom.zero_apply]
+  le_iff_re_im := by simp only [AddMonoidHom.id_apply, AddMonoidHom.zero_apply, and_true,
+    forall_const]
 #align real.is_R_or_C Real.isROrC
 
+noncomputable instance toStarOrderedRing : StarOrderedRing ℝ := by
+  apply StarOrderedRing.ofNonnegIff'
+  intros x y h z
+  simpa [add_le_add_iff_left] using h
+  simp only [star_trivial]
+  intros z
+  refine ⟨ (fun h => ⟨ Real.sqrt z, (Real.mul_self_sqrt h).symm⟩ ), fun hz => ?_ ⟩
+  rw [(Exists.choose_spec hz)]
+  exact mul_self_nonneg _
+
```

Comment:
This one should also be pointless, as the instance below proves a more general case.